Gemfile

```diff
@@ -135,6 +135,8 @@ gem 'geocoder', '1.6.4'
 gem 'parallel'
 gem 'fx'
 gem 'unicode-display_width', '1.6.0'
+gem 'addressable'
+gem 'rss'
```

Comment:
Question: Is this the exact same library we were using which is now not part of the core anymore? I'm asking mostly because the RSS feature is critical for tiplines and I don't remember if in the tests it's mocked.

Reply:
Oh interesting. I think that's the case but will look into to confirm - this was a set of changes made three months ago, before I started upgrading GraphQL, so memory is a little rusty and I don't see a related note in the commit history. Tests are passing so I think it should be okay, but don't want to include things we don't need.

Reply:
Yeah, rss went from being a [default gem in 2.7.0](https://stdgems.org/2.7.0/) to being a [bundled gem in 3.0.+](https://stdgems.org/3.0.6/), which means we have to explicitly declare it

Reply:
Great, thanks!